engine/build.gradle.kts

```
@@ -67,16 +82,22 @@ tasks.named<Javadoc>("javadoc") {
         (options as StandardJavadocDocletOptions).addBooleanOption("html5", true)
     }
 }
+tasks.register<Jar>("testJar") {
+    from(sourceSets.named("junitTest").get().output)
+    classifier = "tests"
+}
+
 publishing {
     publications {
         create<MavenPublication>("mavenJava") {
-            artifactId = "knotx-knot-engine-handler"
+            artifactId = "knotx-knot-engine-core"
```

```suggestion
            artifactId = "knotx-fragments-engine"
```
Rest of the modules and their description should be also updated.